app/preview/ui.test.js

```diff
@@ -123,13 +117,12 @@ test('when existing marketing data - The client application type section and bro
   const minimumResolutionRow = browserBasedExpandaleSectionTable.find('[data-test-id="view-section-table-row-minimum-desktop-resolution"]');
   const hardwareRequirementsDescriptionRow = browserBasedExpandaleSectionTable.find('[data-test-id="view-section-table-row-hardware-requirements-description"]');
   const additionalInformationRow = browserBasedExpandaleSectionTable.find('[data-test-id="view-section-table-row-additional-information"]');
-
   await t
     .expect(clientApplicationTypesSection.exists).ok()
     .expect(clientApplicationTypesSection.find('h3').innerText).eql('Client application type')
 
     .expect(browserBasedExpandableSection.exists).ok()
-    .expect(browserBasedExpandableSection.innerText).eql('Browser based application')
+    .expect(browserBasedExpandableSectionText.trim()).eql('Browser based application')
```

Comment:
might need to do the rest of the innerText expects after this one as they might fail after this one passes